### ISSUE TITLE
FORCE now incorporates download of Landsat and Sentinel-2 from the cloud!

### ISSUE THREAD

**Issue by wandadk** [RETURN-project/makeDataCube#23]
The last version of FORCE has new functionality to download the Landat AND Sentinel-2 data from the Cloud (google cloud). This can simplify the makeDataCube code a lot! See https://force-eo.readthedocs.io/en/latest/howto/level1-csd.html

**Comment by wandadk** [comment]
@PabRod : I think it is worth exploring this new functionality, since this will allow to remove our Python functions from  the R package. In addition, we can also easily download Sentinel data for the test cubes.

**Comment by PabRod** [comment]
Hi @wandadk,

just a warning here: the branch you used for exploring this ([feature/FORCEcsd](https://github.com/RETURN-project/makeDataCube/tree/feature/FORCEcsd)) branches from a slightly old version of master. Perhaps you forgot to pull first. 

I've fixed it with [this commit](https://github.com/RETURN-project/makeDataCube/commit/116527a4bbb715d85bf40285282b798c41ff5cb4). I think I've fixed it cleanly, but it will be a good idea to check twice that the merge didn't destroy anything in the vignette.

If you see anything strange, let me know or, worst-case scenario, just revert the merge (the abovementioned commit https://github.com/RETURN-project/makeDataCube/commit/116527a4bbb715d85bf40285282b798c41ff5cb4).

Cheers!
